trie/iterator_test.go

```diff
@@ -612,5 +671,33 @@ func TestIteratorNodeBlob(t *testing.T) {
 	}
 	if count != len(found) {
 		t.Fatal("Find extra trie node via iterator")
+
+	}
+}
+
+// isTrieNode is a helper function which reports if the provided
+// database entry belongs to a trie node or not. Note in tests
+
+// only single layer trie is used, namely storage trie is not
+// considered at all.
+func isTrieNode(scheme string, key, val []byte) (bool, []byte, common.Hash) {
+	var (
+		path []byte
+		hash common.Hash
+	)
+	if scheme == rawdb.HashScheme {
+		ok := rawdb.IsLegacyTrieNode(key, val)
+		if !ok {
```

Comment:
@Francesco4203 Could u add some unittest specific for IsLegacyTrieNode and IsAccountTrieNode?

Reply:
Added